src/utils.ts

```diff
@@ -150,3 +151,17 @@ export function parseOptions<T>(
         undefined,
     ];
 }
+
+export function fileName(path: string): string {
+    return path.substr(-5).toLowerCase() === '.json' ? path : path + '.json';
+}
+
+export function fileError(error: NodeJS.ErrnoException | null): void {
+    if (error) throw new Error('[interface-forge] ' + JSON.stringify(error));
+}
+
+export function fileExists<T>(filename: string): T | null {
+    if (fs.existsSync(filename))
+        return JSON.parse(fs.readFileSync(filename, 'utf-8')) as T;
```

Comment:
i dont think you need to specify the encoding since utf-8 is default

Reply:
you are right, but passing "utf-8" as a param will force the return value to be a string (otherwise it could also be a blob)